docs/02-providers/aws/README.md

```diff
@@ -23,6 +23,11 @@ provider:
   stage: dev # Set the default stage used. Default is dev
   region: us-east-1 # Overwrite the default region used. Default is us-east-1
   variableSyntax: '\${{([\s\S]+?)}}' # Overwrite the default "${}" variable syntax to be "${{}}" instead. This can be helpful if you want to use "${}" as a string without using it as a variable.
+  stackTags: # Optional CF stack tags
+   key: value
+  stackPolicy: # Optional CF stack policy. The example below allows updates to all resources except deleting/replacing EC2 instances (use with caution!)
+    - {Effect: Allow, Principal: "*", Action: "Update:*", Resource: "*"}
+    - {Effect: Deny, Principal: "*", Action: [Update:Replace, Update:Delete], Condition: {StringEquals: {ResourceType: [AWS::EC2::Instance]}}}
 ```
```

Comment:
could you arrange this into proper yaml? I know this is valid, but we really want the examples to be as clean as possible.